File: app/src/main/java/io/mgba/UI/Fragments/Settings/VideoFragment.java
package io.mgba.UI.Fragments.Settings;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import io.mgba.R;

public class VideoFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.video_settings);
    }

}

File: app/src/main/java/io/mgba/ui/fragments/settings/VideoFragment.kt
package io.mgba.ui.fragments.settings

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import io.mgba.R

class VideoFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(savedInstanceState: Bundle, rootKey: String) {
        addPreferencesFromResource(R.xml.video_settings)
    }

}
